### PR TITLE
maint: styles for dark theme search bar

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
image
image

background is blurred when search is active

### CODE REVIEW

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/fuse-search.css [hunk 1]
```diff
@@ -160,3 +161,8 @@ html[data-theme="light"] .highlight {
 .search-icon {
   cursor: pointer;
 }
+
+.bd-main.blurred {
+  opacity: 0.5;
+  transition: opacity 0.3s ease;
+}
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/fuse-search.css:168]: Nice!

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/js/fuse_extension.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 const searchBar = document.getElementById("search-bar");
 const searchBarBox = searchBar.querySelector(".bd-search input");
 resultsContainer = document.getElementById("results");
+const content = document.querySelector(".bd-main");
 
 // Function to expand the search bar and display results
 function expandSearchBox() {
   searchBarBox.classList.add("expanded");
+  // content.style.opacity = 0.5;
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/js/fuse_extension.js:9]:
```suggestion
```

**Review by jorgepiloto — APPROVED** [review]
Working nice and solid! Let's merge this.

Some style tasks that we should address include:

- Restoring the right margin between the search bar and the icons

**Review by jorgepiloto — APPROVED** [review]

**Comment by Revathyvenugopal162** [comment]
> Working nice and solid! Let's merge this.
> 
> Some style tasks that we should address include:
> 
> * Restoring the right margin between the search bar and the icons

sure i will add these in another PR

**Pull request merged**